gateway/1.8/modules/ROOT/pages/flex-architecture-high-availability.adoc

```diff
@@ -0,0 +1,94 @@
+== Implementation 1: High Availability with the Same APIs in Two Separate Regions
+
+Implementation 1 provides high availability for non-region specific traffic. The DNS service is configured for active-active failover meaning each region services traffic at all times.
+
+[[implementation-1a]]
+=== Implementation 1A: Cross-Regional Flex Gateway
+
+In this implementation:
+
+* A single Flex Gateway has Flex Replicas distributed across multiple region.
+* The Flex Gateway is registered once in Anypoint Platform and deployed across multiple regions and availability zones.
+* Logs and metrics for each Flex Gateway are consolidated in Anypoint Platform.
+* The DNS service is configured for active-active failover to direct traffic to any region at all times.
+* APIs deployed to the Gateway are available in all regions and availability zones. 
+* API configurations including upstream services and policies are the same for all regions. Upstream services must be region-agnostic meaning either the DNS is local or it lacks any reference to the region.
+* The Redis cluster should be replicated across regions or Redis shared storage can be used. Redis replication is required when traffic is not region specific.   
+* The only cross-regional traffic is Redis synchronization.
+
+In the following diagram, Flex Replicas in different regions and availability zones have distinct Kubernetes deployments:
+
+image:Flex-Gateway-DR-1.png["A detailed view of implementation 1A, which contains the necessary services to support High Availability in different regions Flex Replicas."]
+
+
+[[implementation-1b]]
+=== Implementation 1B: Region-Specific Flex Gateway
+
+In this implementation:
+
+* Each region has a distinct Flex Gateway with all resources as independent entities.
+* A separate Flex Gateway is registered for each region.
+* Logs and metrics are specific to their region because each Flex Gateway is a distinct entity in Anypoint Platform.
+* The DNS service is configured for active-active failover to direct traffic to any region at all times.
+* API deployments are specific to each Flex Gateway. For each intended API a region supports, you must deploy an API instance to the region's Flex Gateway. 
+* API Groups can share contracts (access requests) across regions.
+* Shared storage does not require synchronization. 
+* There is no cross-regional traffic.
+
+In the following diagram, all APIs are duplicated and do not require identical configurations:
+
+image:Flex-Gateway-DR-2.png["A detailed view of implementation 1B, which contains the necessary services to support High Availability in region-specific Flex Gateways."]
+
+[[implementation-2]]
+== Implementation 2: Disaster Recovery for APIs
+
+This implementation is similar to <<implementation-1a>>. However, the DNS service is active-passive, so the standby region only receives traffic if the primary region becomes unhealthy. This implementation is less complex and cheaper to configure than implementation 1a.
+
```

Comment:
Cheaper is relative here. I would only say less complex.

In this case there is a whole region that is idle (not serving traffic) so I wouldn't call this 'cheaper'